manifests/s/SanfordLP/DYMOConnect/1.4.2/SanfordLP.DYMOConnect.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: SanfordLP.DYMOConnect
+PackageVersion: 1.4.2
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://s3.amazonaws.com/download.dymo.com/dymo/Software/Win/DCDSetup1.4.2.82.exe

```

```suggestion
  InstallerUrl: https://download.dymo.com/dymo/Software/DYMO%20Connect%201.4.2/DCDSetup1.4.2.82.exe
```
I got this link from [here](https://www.dymo.com/en_CA/online-support-compatibility-chart). Would it help with URL validation?

---

Hi @DinCahill ,
Thanks for the suggestion, but I found that wingetcreate gave me the following error when I used download.dymo.com url :

```
Downloading and parsing: https://download.dymo.com/dymo/Software/DYMO Connect 1.4.2/DCDSetup1.4.2.82.exe...
Failed to download installer.
HTTP response was unsuccessful. Status code: Forbidden
```
This is why I used the s3.amazonaws.com link

---

Ah, I see. It's hitting a Cloudflare CAPTCHA page. winget-cli can download it, but wingetcreate and YamlCreate can't. Thanks for the reply.

---

If the dymo.com installer url will resolve in winget cli installations, we should use that url rather than the direct to cdn link even if wingetcreate fails.

---

I see your reasoning, but I disagree. We know that the download.dymo.com link cannot be used by some products. I feel that we can therefore assume it could possibly break other downstream products such as winstall, winget.run, wingy, etc. Hence my preference for the more universally accessible s3.amazonaws.com link.

---

@jamespik @DinCahill : What are your thoughts on this matter?

---

@DW-42, since this is a community repository, we'd want the installer URL to align with the publisher's installer url (from dymo.com). As long as the url still works for winget-cli installations, we believe that is the right decision so that we can maintain the consistency of these packages based on the publisher's intended installer url.  Feel free to file a bug on Winget-Create so that we can also improve our tool and resolve this issue. Thanks!

---

Ah-ha! I see.